src/api/app/models/event/relationship.rb

```diff
@@ -25,5 +25,9 @@ def notifiable_id
 
       nil
     end
+
+    def originator
+      payload_address('who')
```

Comment:
This is the same [Event::Request](https://github.com/openSUSE/open-build-service/blob/842c83e03dc0e48516d68595a296ae2dfae0ee0c/src/api/app/models/event/request.rb#L18-L20), overriding what is defined in [Event::Base#originator](https://github.com/openSUSE/open-build-service/blob/842c83e03dc0e48516d68595a296ae2dfae0ee0c/src/api/app/models/event/base.rb#L210-L214).